Documentation/VSCodeConfiguration.md

```diff
@@ -10,14 +10,17 @@ Both C++ comprehension tools listed below report fake errors.
 
 ### clangd
 
-The official clangd extension can be used for C++ comprehension. You'll have to use the following .clangd:
+The official clangd extension can be used for C++ comprehension. It is recommended in general, as it is most likely to work on all platforms. You'll have to use the following .clangd:
 
 ```yaml
 CompileFlags:
+  Add: [-D__serenity__, -I/path/to/serenity/Toolchain/Local/i686/i686-pc-serenity/include/c++/11.2.0, -I/path/to/serenity/Toolchain/Local/i686/i686-pc-serenity/include/c++/11.2.0/i686-pc-serenity]
```

Comment:
Maybe add a note that this line is only required when using the gcc toolchain
The Clang one adds these in the compile commands